packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -192,14 +192,14 @@ export default class EthereumAnchorService implements AnchorService {
           message: json.message,
         }
       case 'COMPLETED': {
-        const { anchorRecord } = json
```

Comment:
This change would make js-ceramic incompatible with [ceramic-anchor-service](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/develop/src/models/request-presentation.ts#L30). I would advice adding a corresponding PR to ceramic-anchor-service repository to add `anchorCommit` field in addition to the present `anchorRecord` one. Please note: _add_, not _rename_. We want to maintain backwards compatibility.